### PR TITLE
Add Hackling Bot

### PR DESCRIPTION
**Pull request by hackling**
As title describes

### CODE REVIEW

**Comment by hackling** [comment]
@davekinkead This one has been included in the `players.rb`. It's a prime candidate for merging.

**Comment by davekinkead** [comment]
Keep playing those cards close to your chest @hackling

**Comment by hackling** [comment]
I'm actually being serious with the following question:

What's wrong with Hackling bot that is preventing it from being included? It's obvious why I couldn't have TroubleMaker, but Hackling bot isn't at all malicious, and execution of it won't effect the execution of any of the other bots provided.

File: lib/bots/hackling.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require "#{
```
Comment (ghiculescu) [inline lib/bots/hackling.rb:1]: out of curiosity, is there a reason you need to put this code in a require statement?

i _could_ download it and test and figure it out myself but... yeah.

Comment (hackling) [inline lib/bots/hackling.rb:1]: It's randomly selecting a bot to inherit from. To keep with the current pattern, I'm doing the random selection as part of the parameter passed to `require`, so I can require the right bot.

**Comment by davekinkead** [comment]
It's fine and works pretty well when it clones the right victim. I've merged into the submissions branch.

File: lib/bots/hackling.rb (hunk 1 above)
Comment (hackling) [inline lib/bots/hackling.rb:1]: That description is crap.

Anyway, it doesn't really. I was just doing something else, with it, got to this point, and never bothered refactoring.

**Pull request closed without merging**